pages/4_AWS_Health.py

```
@@ -43,9 +68,30 @@
+
+sender_email = st.text_input("Sender Email (Verified in SES)")
+recipient_email = st.text_input("Recipient Email")
+
+if st.button("Fetch Events and Notify"):
+    try:
+        get_all_events_and_notify(df, sender_email, recipient_email)
+        st.success("Notifications sent for all scheduledChange events!")
+    except ValueError as e:
+        st.error(str(e))
+
+bucket_name = st.text_input("Enter S3 bucket name:")
```

Ruff (0.8.2)

`st` may be undefined, or defined from star imports

(F405)